model/notification/db.go

```diff
@@ -152,6 +152,12 @@ func FindByEventID(id string) ([]Notification, error) {
 	return notifications, err
 }
 
+func FindUnprocessed() ([]Notification, error) {
+	notifications := []Notification{}
+	err := db.FindAllQ(Collection, db.Query(bson.M{sentAtKey: bson.M{"$exists": false}}), &notifications)
+	return notifications, err
```

Comment:
wrap the error? 

do we want to nil if not found?

Reply:
There was an in-person discussion around this question. This test
```
func (s *notificationSuite) TestingNotFound() {
	s.NoError(db.Clear(Collection))
	unprocessedNotifications, err := FindUnprocessed()
	s.NoError(err)
	s.Len(unprocessedNotifications, 0)
}
```
demonstrated that the FindAllQ returned no error 
```
func FindUnprocessed() ([]Notification, error) {
	notifications := []Notification{}
	err := db.FindAllQ(Collection, db.Query(bson.M{sentAtKey: bson.M{"$exists": false}}), &notifications)

	return notifications, err
}
```

Reply:
wrapped the error